doc/scripts/write_script_help.py

```diff
@@ -0,0 +1,38 @@
+"""
+Dynamically build the rst documentation with the script help text.
```

Comment:
Nice!  Feel free to add in any other script you have the
endurance to handle.

Reply:
Incoming...